documentation: modified testing.md for clarity

Infrasaurus opened the pull request:
Line 69: Changed punctuation position for readability
Line 71: Changed case of "clairv4-org" for consistency with Quay limitations
Lines 84-87: Added clarity on viewing results for those unfamiliar with Quay.

Addresses #1180

ldelossa commented:
Thanks for this @justin-sysadmin - we have ppl bump into this all the time so the doc update is very useful.

Couple thing with submitting PR's to our repository. 

You must sign off on your PR. You can do this with "git commit --amend -s" 

We also force a commit structure of
```
[subject]: [summary]

[body]
```

You can fix both issues at once with the git command above, if you are not familiar.

ldelossa commented:
whoops, you passed the commit check pass fine, sorry just assumed that is why this failed. you just need to sign off on the commit.

Infrasaurus committed:
documentation: modified testing.md for clarity

Line 69: Changed punctuation position for readability
Line 71: Changed case of "clairv4-org" for consistency with Quay limitations
Lines 84-87: Added clarity on viewing results for those unfamiliar with Quay.

Addresses #1180

Signed-off-by: justin-systadmin <justingn@gmail.com>

Infrasaurus commented:
Thanks for the patience!  First **ever** commit, had to crash course the CLI on my Mac to get the signoff in place.  Checks look good!
